src/property.ts

```diff
@@ -214,7 +215,28 @@ export class Property implements PropertyPath {
 
 			// Set
 			if (typeof options.set === "function") {
+				const property = this;
 				this.changed.subscribe(function(e) { options.set.call(this, e.newValue); });
```

Comment:
This was prompted by an issue where back reference properties on section items were not set correctly, since the new `init` behavior for sections does not technically "change" the property.

Reply:
Combined the property `changed` subscription with the rule.